src/views/Home/components/Milestones/index.tsx

```diff
@@ -2,37 +2,42 @@ import * as React from "react"
 import "./index.scss"
 import { MilestoneProps } from "Props/props"
 import milestoneResults from "Props/milestones/milestones"
-import milestonesBorderMobile from "assets/milestones/milestone_header_mobile.svg"
+import { ReactComponent as BorderMobile } from "assets/milestones/milestone_header_mobile.svg"
 
 const Milestones: React.FC = () => (
   <div className='milestones-component'>
-    <div className='milestones-component__header'>
-      <div className='milestones-component__header-circle'>
-        <div className='milestones-component__header-text'>
-          Milestones of 2021
+    <div className='milestones-component__container'>
+      <div className='milestones-component__header'>
+        <div className='milestones-component__header-circle'>
+          <div className='milestones-component__header-text'>
+            Milestones of 2021
+          </div>
+          <BorderMobile className='milestones-component__header-border' />
         </div>
+        <div className='milestones-component__header-ellipse' />
       </div>
-      <div className='milestones-component__header-border'>
-        <img src={milestonesBorderMobile} alt='error' />
-      </div>
-      <div className='milestones-component__header-ellipse' />
-    </div>
 
-    <div className='milestones-component__achievements'>
-      {milestoneResults.map(
-        ({ image, milestoneName, milestoneStat, id }: MilestoneProps) => (
-          <div className='milestone' key={id}>
-            <div className='milestone__image'>
-              <img src={image} alt='error' />
-            </div>
-            <div className='milestone__box'>
-              <div className='milestone__stat'> {milestoneStat}</div>
-              <div className='milestone__name'> {milestoneName}</div>
+      <div className='milestones-component__achievements'>
+        {milestoneResults.map(
+          ({ image, milestoneName, milestoneStat, id }: MilestoneProps) => (
+            <div className='milestones-component__milestone' key={id}>
+              <div className='milestones-component__milestone__image'>
+                <img src={image} alt='error' />
+              </div>
```

Comment:
If we want we can probably make this a `ReactComponent`  in the props file when and that should hopefully help with consistency.

Reply:
I tried that but I don't know a way to pass the className to the component. I suppose we could make `image` a type and make the component here with `React.createComponent()`, but that seems to complicate things.